server.go

```diff
@@ -3658,14 +3658,13 @@ func (s *server) ConnectToPeer(addr *lnwire.NetAddress,
 	// connection.
 	if reqs, ok := s.persistentConnReqs[targetPub]; ok {
 		srvrLog.Warnf("Already have %d persistent connection "+
-			"requests for %x@%v, connecting anyway.", len(reqs),
-			targetPub, addr)
+			"requests for %v, connecting anyway.", len(reqs), addr)
 	}
 
 	// If there's not already a pending or active connection to this node,
 	// then instruct the connection manager to attempt to establish a
 	// persistent connection to the peer.
-	srvrLog.Debugf("Connecting to %x@%v", targetPub, addr)
+	srvrLog.Debugf("Connecting to %v", addr)
```

Comment:
I think you need to use `%s` to print it as a string.

Reply:
works with `%v` too :)

Reply:
well it does work, because currently it shows the full string via `%v` ... unfortunately, I'm not a Rust pro, yet, so I can't tell what is best practice .... should I change it to `%s` or keep as is? @yyforyongyu